### ISSUE TITLE
body-parser and express module cannot read manifest.json (GET http://localhost:3001/public/manifest.json 404 (Not Found) manifest.json:1 Manifest: Line: 1, column: 1, Unexpected token.)

### ISSUE THREAD

**Issue by indunie** [expressjs/body-parser#349]
I am using Node.js Version 10.14.1 on windows 10
with express and body-parser

 I am creating a basic react fronted with a node backend. I setup mysql database connection and all set. I want to insert user details using /createUser api call to store data to database. I run the server/app.js, react serve and index.js which contained my listener for /createUser. Once I run the listener, It is giving the error below.

GET http://localhost:3001/public/manifest.json 404 (Not Found) manifest.json:1 Manifest: Line: 1, column: 1, Unexpected token.
//my listener
```
const express = require('express');
const bodyParser = require('body-parser');
const store = require('./store');
const app = express();
app.use(express.static('public'));

app.use(bodyParser.urlencoded({ extended: true }));
app.use(bodyParser.json());

app.post('/createUser', (req, res) => {
    console.log(req.body);
    store
        .createUser({
            username: req.body.username,
            password: req.body.password
        })
        .then(() => res.sendStatus(200))
})
app.listen(3001, () => {
    console.log('Server running on http://localhost:3001')
})
```
full source : [https://github.com/indunie/tms2](url)

How do I fix this issue?

**Comment by dougwilson** [comment]
Duplicate of https://github.com/expressjs/express/issues/3881

**Comment by indunie** [comment]
solved [[expressjs/express#3881](https://github.com/expressjs/express/issues/3881)](url)
